lib/plugins/helper/open_graph.js

```diff
@@ -38,8 +43,9 @@ function openGraphHelper(options) {
   var page = this.page;
   var config = this.config;
   var content = page.content;
+  content = removeScripts(content);
```

Comment:
It should be sufficient to use `util.stripHTML` here since most regular expression on tag replacement is error prone, see http://stackoverflow.com/a/535022.

It will print inline scripts content on the description on certain circumstance but we should not be too smart on that.

Also L46 can be combined with L45 for brevity.